### PR TITLE
feat(sig-services): Add customer zero guideline

### LINKED ISSUE #159: [SIG Services][Guideline] Have a customer 0
Before releasing publicly consider getting a customer 0 so you can get instant feedback on the service before users start complaining right after initial release. Gather UX feedback, don't share any insider info with the customer, let them use it but watch them closely and be in contact as a close support.

Not a strong requirement, rather a nice to have.

### PR DESCRIPTION
**Pull request by mimotej**
Resolves #159

### CODE REVIEW

File: sig-services/service-requirements/customer-zero.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Have a customer zero
+persona: Service developer
+phase: Service development
+---
+
+**This requirement is optional**
+
+Having customer zero allows service to get instant feedback before the public release. From customer zero, developers can gather feedback on features of their service, UX design, and others. An important part of this process is not to share any insider information about the project with customer zero. Customer should be allowed to use the service on their own but should be watched by developers and in close contact with support.
```
Comment (Shreyanand) [inline sig-services/service-requirements/customer-zero.md:9]:
```suggestion
Having customer zero allows service to get instant feedback before the public release. From customer zero, developers can gather feedback on features of their service, UX design, and others. An important part of this process is not to share any developer or maintainer knowledge about the project with customer zero. Customer should be allowed to use the service on their own but should be observed by developers and in close contact with support.
```

File: sig-services/service-requirements/customer-zero.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Requirements
+
+- Customer zero is established; customer zero should not have any insider knowledge of the project and should be allowed to use the service on their own
+
+## Reasoning
+
+Using customer zero leads to a better quality of the service before release. Developers can find major flaws in their UX design or problems with the concepts of their service.
```
Comment (Shreyanand) [inline sig-services/service-requirements/customer-zero.md:17]:
```suggestion
Identifying customer zero leads to a better testing of the service before release. Developers benefit from this testing and can find major flaws in their UX design or problems with the concepts of their service.
```

File: sig-services/service-requirements/customer-zero.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+phase: Service development
+---
+
+**This requirement is optional**
+
+Having customer zero allows service to get instant feedback before the public release. From customer zero, developers can gather feedback on features of their service, UX design, and others. An important part of this process is not to share any insider information about the project with customer zero. Customer should be allowed to use the service on their own but should be watched by developers and in close contact with support.
+
+## Requirements
+
+- Customer zero is established; customer zero should not have any insider knowledge of the project and should be allowed to use the service on their own
```
Comment (Shreyanand) [inline sig-services/service-requirements/customer-zero.md:13]:
```suggestion
- Customer zero should be identified
- Customer zero should not have any developer or maintainer knowledge of the project and should be allowed to use the service on their own
```

**Review by Shreyanand — COMMENTED** [review]
small nits, otherwise /lgtm

File: sig-services/service-requirements/customer-zero.md (hunk 2 above)
Comment (mimotej) [reply sig-services/service-requirements/customer-zero.md]: Fix in latest commit

File: sig-services/service-requirements/customer-zero.md (hunk 3 above)
Comment (mimotej) [reply sig-services/service-requirements/customer-zero.md]: Changed in latest commit good point :+1:

**Review by Shreyanand — APPROVED** [review]
/lgtm

**Pull request merged**